CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+Here is some information that's good to know when contributing to JavaParser.
+
+- if you're new and like to contribute something, check out the [easy issues](https://github.com/javaparser/javaparser/labels/Easy)
+- we work on JavaParser because we like to, not because it earns us money. Please remember that we try to run a professional project in our spare time, on a budget of zero.
+- if you know how to fix a problem, please fix it and open a pull request instead of opening an issue.
+- a pull request should include tests. You can either use BDD (JBehave) or JUnit.
+- there is some interesting information on the [wiki](https://github.com/javaparser/javaparser/wiki).
```

Comment:
We're mentioning BDD and the coding guidelines explicitly, do you think there's more info that's essential for committers?